account_analytic_category/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright: Henrik Norlin
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Analytic Categories",
+    "version": "16.0.1.0.0",
+    "author": "Henrik Norlin, Odoo Community Association (OCA)",
+    "category": "Account",
+    "website": "https://github.com/OCA/account-analytic",
+    "depends": [
+        "account_financial_report",  # account.move.line.analytic_account_ids
```

Comment:
I thought this dependency was a bit heavy.  Maybe the part that depends on `account.move.line.analytic_account_ids` can be handled in a separate glue module in the account-financial-reporting repo.

Reply:
I made a new module `account_move_line_analytic_account_ids`.

Reply:
Nice :) I have added `analytic_mixin_analytic_account` as a dependeny.